tests/unit/test_calculate.py

```diff
@@ -261,7 +266,10 @@ def test_commuting_car(
 ):
     """Test co2 calculation for commuting by car"""
 
-    # Calculate co2 emissions
+    # NOTE: This is more of a functional test.
+    # If it's supposed to stay a unit test, we should mock the `calc_c2_...` methods
+    # and check if they're called!
+
```

Comment:
imo, we can move it to functional tests

and maybe just add a unit test for `commuting_emissions_group`?

Reply:
We'll do that, in a separate branch! Usually I would create an issue right away but I am expecting to iterate over that code in the next days again and again .. so a NOTE does the job too :)